src/csync/csync_exclude.cpp

```diff
@@ -319,29 +319,31 @@ bool ExcludedFiles::isExcluded(
     const QString &basePath,
     bool excludeHidden) const
 {
+    const QFileInfo fileInfo(filePath);
+    if (!fileInfo.exists(filePath)) {
+        return true;
```

Comment:
Here I have the question what the actual correct behavior for non exisitng files is.
With the backup scenario I discovered that we are running expansive actions like isDir on files that don't even exist.